app/liquid_tags/codepen_tag.rb

```diff
@@ -1,20 +1,22 @@
 class CodepenTag < LiquidTagBase
   PARTIAL = "liquids/codepen".freeze
   REGISTRY_REGEXP =
-    %r{\A(http|https)://(codepen\.io|codepen\.io/team)/[a-zA-Z0-9_\-]{1,30}/pen/([a-zA-Z0-9]{5,32})/{0,1}\z}
+    %r{\A(http|https)://(codepen\.io|codepen\.io/team)/[a-zA-Z0-9_\-]{1,30}/(pen|pen/preview)/([a-zA-Z0-9]{5,32})/{0,1}\z}
```

Comment:
Fringe case, but some users may grab their link from the Codepen "Embed" modal. This link is of the format:
```ruby
https://codepen.io/msarit/embed/eYJZdmp?default-tab=html%2Cresult&editable=true
```

I think we should add `embed` in the third capture group `(pen|pen/preview)`.

Also I am not sure if it is worth it in this PR to handle possible url params like we see in the example link above. Perhaps that can be a separate PR. 😅

Reply:
Great call, updating it to `/(pen|embed)(/preview)?/` and I'll add tests to cover those too

parsing the params would be a nice improvement for ease of use... agreed with doing that another time though 😅

Fortunately with this suggested change, copying the full embed url from codepen and manually replacing `?` and `&` with spaces is the only translation the author needs to do for it to work in the liquid tag. So it's not toooo much overhead now 💜